marlon/simulate.py

```diff
@@ -49,10 +49,10 @@ def run_simulation(iteration_count):
         assert np.shape(reward) == ()
 
         # If there is a jump in the reward for this step, record it for UI display.
-        if reward != 0 or iteration == iteration_count-1:
-            simulation.append(generate_graph_json(gym_env, iteration+1))
         if done:
-            break
+            simulation.append(generate_graph_json(gym_env, iteration_count))
```

Comment:
Shouldn't this still break the loop if done? What happens when the loop continues after `wrapped_env.step` returns 'done'? Or alternatively what does 'done' actually mean?

Reply:
Oops, yes a break is needed after the simulation.append. I had to redo the commit, and forgot to add the break. Good catch